NEWS

```diff
@@ -1,3 +1,32 @@
+uftrace v0.9.4
+--------------
+* dynamic tracing update
+  improve success rate on x86_64
+  dynamic unpatch on x86_64 (for -mfentry or -mrecord-mcount)
+  add -U/--unpatch option for dynamic tracing
+  experimental support for aarch64
+
+* script update
+  luaJIT (lua 5.1) support by Byeonggon
+  python3 support
+
+* build change
+  update configure script for better compatibility
+  handle common cross compile settings
+
```

Comment:
Maybe we can add the following description.
```suggestion

* tui change
  add 's' key to sort by the next column in tui report mode
  use --report option to start tui with report mode

```

Reply:
Done